web/server/README.md

```diff
@@ -91,6 +91,28 @@ Netdata supports access lists in `netdata.conf`:
 
 - `allow management from` checks the IPs to allow API management calls. Management via the API is currently supported for [health](../api/health/#health-management-api)
```

Comment:
We need a long comment here, telling us how to configure the various ports for SSL in the various cases

Reply:
When I updated the documentation, it was necessary to move the SSL section from the previous position, because it was behind the ACL section, I moved it and I created the necessary documentation .